diff --git a/node_modules/vite-plugin-ssr/dist/cjs/node/runtime/renderPage/getPageAssets.js b/node_modules/vite-plugin-ssr/dist/cjs/node/runtime/renderPage/getPageAssets.js
index 8e6d023..d6af212 100644
--- a/node_modules/vite-plugin-ssr/dist/cjs/node/runtime/renderPage/getPageAssets.js
+++ b/node_modules/vite-plugin-ssr/dist/cjs/node/runtime/renderPage/getPageAssets.js
@@ -83,7 +83,7 @@ function resolveClientEntriesDev(clientEntry, viteDevServer, configVps) {
         // VPS client entry
         (0, utils_1.assert)(clientEntry.endsWith('.js'));
         const req = require; // Prevent webpack from bundling client code
-        const res = req.resolve;
+        const res = typeof Bun != "undefined" ? (toPath) => Bun.resolveSync(toPath, __dirname) : req.resolve
         try {
             // For Vitest
             // [RELATIVE_PATH_FROM_DIST] Current file: node_modules/vite-plugin-ssr/node/runtime/html/injectAssets.js
